### ISSUE TITLE
updating prelude throws errors of not finding files or directories

### ISSUE THREAD

**Issue by kirk86** [bbatsov/prelude#1132]
Is this a normal behavior every time you update prelude to get the following error messages?

^L                                                                    
Compiling file /Users/jm/.emacs.d/modules/prelude-go.el at Tue Oct 24 11:55:01 2017
Entering directory ‘/Users/jm/.emacs.d/modules/’                      
prelude-go.el:40:1:Error: Cannot open load file: No such file or directory, go-projectile
^L                                                                    
Compiling file /Users/jm/.emacs.d/modules/prelude-ocaml.el at Tue Oct 24 11:55:02 2017
prelude-ocaml.el:54:1:Error: Cannot open load file: No such file or directory, tuareg
^L                                                                    
Compiling file /Users/jm/.emacs.d/modules/prelude-web.el at Tue Oct 24 11:55:02 2017
prelude-web.el:37:1:Error: Cannot open load file: No such file or directory, web-mode

**Comment by ghaskins** [comment]
I see this same behavior

**Comment by bbatsov** [comment]
It's not normal, but it's also not a big problem. How do you update exactly?

**Comment by kirk86** [comment]
Usually I update the packages through package-list.
Then I use prelude-update, or sometimes might update the packages directly from prelude through prelude-update-packages.

**Comment by pajatopmr** [comment]
I see this behavior as well manifested in the source file `prelude-ivy.el`. Out of curiosity I started reading that library and saw that there was a warning associated with "here" in the last line, specifically that `prelude-required-packages` was not known to be defined. I recognize that this is not exactly devastating but I am curious to understand better how to get a symbol "known to be defined".  I tried a few things, like `(require 'prelude-packages)` but only moved the harmless warning around. For those that care, I posted this question to [Emacs Stack Exchange](https://emacs.stackexchange.com/questions/43696/how-to-get-emacs-to-know-that-a-symbol-is-defined)
